### PR TITLE
Docs: Add link to pages in contribution guide

### LINKED ISSUE #982: docs: Create contribution guide
Many open source projects include contribution guides (usually in a CONTRIBUTING.md file) to help new contributors make their first commit. It might be a good idea to add such a file to the zx repository.

### PR DESCRIPTION
**Pull request by LifeforDream**
Fixes #982

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
-# How to Contribute
+# Contribution Guide
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
+## Disclaimer
+
+This is a snapshot of a contribution guide.
```
Comment (antongolub) [inline docs/contributing.md:5]:
```md
# Contribution Guide

zx is a fully [open-source project](https://github.com/google/zx), which is developing by the community for the community. 
We welcome contributions of any kind, including but not limited to:
* Bug reports
* Feature requests
* Code contributions
* Documentation improvements
* Discussions

https://google.github.io/zx/contribution
```

**Review by antongolub — CHANGES_REQUESTED** [review]

File: docs/contributing.md (hunk 1 above)
Comment (LifeforDream) [reply docs/contributing.md:5]: Moved the link, removed the disclaimer

File: docs/contributing.md [hunk 2]
```diff
@@ -15,14 +32,28 @@ You generally only need to submit a CLA once, so if you've already submitted one
...
+  * Ensure you have Node.js 20+ installed.
+  * Bash is essential for running zx scripts. Linux and macOS users usually have it installed by default. Consider using [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install) or [Git Bash](https://git-scm.com/downloads) if you are on Windows.
+* Fork [the repository](https://github.com/google/zx).
+* Create a new branch.
+* Make your changes.
+  * If you are adding a new feature, please include additional tests. The coverage threshold is 98%.
+  * Create a [conventional-commits](https://www.conventionalcommits.org/en/v1.0.0/) compliant messages.
+* Ensure that everything is working:
+  * `npm run fmt` to format your code.
+  * `npm run cov` to run the tests.
```
Comment (antongolub) [inline docs/contributing.md:47]: npm run cov  > npm test

**Review by antongolub — CHANGES_REQUESTED** [review]

File: docs/contributing.md (hunk 2 above)
Comment (easymikey) [reply docs/contributing.md:47]: `npm run cov` not exist. Maybe `npm run test:coverage` ?

Comment (LifeforDream) [reply docs/contributing.md]: Thanks!
Changed that

**Review by antongolub — APPROVED** [review]
lgtm. Thanks for the contribution!

**Pull request merged**